### PR TITLE
chore: update to bee 1.5.0-rc4-dda5606e

### PR DESCRIPTION
**Pull request by bee-worker**
Updated Bee version 1.5.0-rc1-32c0771c

### CODE REVIEW

**Comment by AuHau** [comment]
We will have to investigate this change in Bee-js: https://github.com/ethersphere/bee/pull/2821

**Comment by bee-worker** [comment]
## 🗒 Bee 1.5.0-rc2-95af6f58 Release notes

## Changelog
* 95af6f58 fix: runtime panic (#2836)

File: src/modules/debug/status.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ import getMajorSemver from 'semver/functions/major.js'
 
 // Following lines bellow are automatically updated with GitHub Action when Bee version is updated
 // so if you are changing anything about them change the `update_bee` action accordingly!
-export const SUPPORTED_BEE_VERSION_EXACT = '1.4.3-1213e063'
-export const SUPPORTED_API_VERSION = '2.0.0'
-export const SUPPORTED_DEBUG_API_VERSION = '1.2.1'
+export const SUPPORTED_BEE_VERSION_EXACT = '1.5.0-rc3-89021656'
+export const SUPPORTED_API_VERSION = ' 3.0.0'
+export const SUPPORTED_DEBUG_API_VERSION = ' 2.0.0'
```
Comment (vojtechsimetka) [inline src/modules/debug/status.ts:11]: These extra spaces should also be removed.
```suggestion
export const SUPPORTED_API_VERSION = '3.0.0'
export const SUPPORTED_DEBUG_API_VERSION = '2.0.0'
```

**Comment by vojtechsimetka** [comment]
superseded by #598

**Pull request closed without merging**